### DISCUSSION TITLE
Syncing Icons turns into Text

### DISCUSSION THREAD

**Discussion by uwwwan** [FlorianWoelki/obsidian-icon-folder#472] [Q&A] [answered]
Whenever I try to sync it the Inherited Icon turns into text which is shown in the screenshot. I am using Tabler Icons for it in the Iconize plugin

![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/125205860/9d4204e6-d10e-42e2-a315-f235551fb38f)

**Comment by nikgreen** [comment, marked as answer]
I've had this issue as well.

Normally easily resolved by going into the **Iconize** settings and checking to see if the **Icon Packs** have synced.

If they've not, simply bring them down again and after a few moments (or restart Obsidian) the icon codes, which is what you are seeing, will be replaced with the icon.

You may also want to check that the **[Icons Background Check](https://florianwoelki.github.io/obsidian-iconize/guide/syncing.html#background-checker)** is enabled

![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/101189/fcfe6c5e-a6ed-4c29-ae26-e71cbd78fbb5)

**Comment by Kribensisis** [comment]
I've had this too, and the solution didn't work for me. The background check was enabled and all icon packs were installed. 

Folder icons just don't sync well in this extension.
